### PR TITLE
Update instructions for Docker Desktop

### PR DESCRIPTION
**Pull request by SSPJ**

### CODE REVIEW

**Comment by SSPJ** [comment]
Appears to have been bitten by https://github.com/pa11y/pa11y-ci/issues/128. I'll check by running on my local when I have a free moment.

File: _pages/laptop-setup.md [hunk 1]
```diff
@@ -49,6 +47,16 @@ While you are welcome to customize your laptop, here are some tools that have wo
 [Virtualenvwrapper]: http://virtualenvwrapper.readthedocs.org/en/latest/#
 [Zsh]: http://www.zsh.org/
 
+## Licensed Software
+
+* [Docker] for all your containerization needs
+
+  The docker daemon and docker CLI tools are available on a free license.
+
+  [Docker Desktop] requires a paid license. Ask in #admins-docker in Slack to get a license.
```
Comment (geekygirlsarah) [inline _pages/laptop-setup.md:56]: An easy way to link the Slack channel is replacing the channel with `[#admins-docker](https://gsa-tts.slack.com/app_redirect?channel=admins-docker)` .

**Comment by geekygirlsarah** [comment]
> Appears to have been bitten by [pa11y/pa11y-ci#128](https://github.com/pa11y/pa11y-ci/issues/128). I'll check by running on my local when I have a free moment.

@bklaas didn't you fight this on an issue on the guide in the past? Do you remember what you did to fix it?

**Comment by echappen** [comment]
Yeah pa11y fails intermittently (we need to look into why). You can try running it again and that usually works.

File: _pages/laptop-setup.md (hunk 1 above)
Comment (SSPJ) [reply _pages/laptop-setup.md]: Thank you, Sarah, added!

**Comment by Sgtpluck** [comment]
@echappen @svenaas what do you think of the [options](https://github.com/18F/development-guide/pull/310#issuecomment-1128991334)?

**Comment by echappen** [comment]
For this PR in particular, I say we go with 1) ignore—fixing footnotes across the site should probably be its own issue.

**Pull request merged**